build.sbt

```diff
@@ -11,7 +11,7 @@ lazy val root = Project(libName, file("."))
   .configs(IntegrationTest)
   .settings(
     name := libName,
-    majorVersion := 1,
+    majorVersion := 2,
```

Comment:
Ooh, major version change. SO, I know this is a breaking change for the PTA teams, but I think it would be worth discussing whether we think this qualifies as a major version. Last time we did a major version it was a LOT of architecture changes IIRC, and we don't always major version breaking changes.

Reply:
Hmm, I think going from v0 to v1 is quite special because it's somewhat saying "now we'll really try to avoid breaking changes / api is pretty stable" and until v1 it feels like maybe you can introduce breaking changes in minor versions, but past v1 I wonder if we should switch to bump it whenever there's a breaking change no matter how big or small (though maybe we haven't clearly defined what our API surface is)

Reply:
we've updated the readme to explain that the account menu is currently experimental and being developed outside this repo by PTA so subject to breaking API changes 👍